lib/ansible/module_utils/docker_common.py

```diff
@@ -144,6 +163,10 @@ def __init__(self, argument_spec=None, supports_check_mode=False, mutually_exclu
             required_together=required_together_params,
             required_if=required_if)
 
+        if HAS_DOCKER_MODELS and HAS_DOCKER_SSLADAPTER:
+            self.fail("Cannot have both the docker-py and docker python modules installed installed together as they use the same namespace and "
```

Comment:
Message cleanup needed. Notice `installed installed`.

Reply:
Duplicate "installed" removed.

Reply:
This was merged so fast I didn't put in my $0.02: 'docker' and 'docker-py' are not 'python modules'. They are 'python packages' and should be called that to avoid confusion with Ansible's 'docker modules'.

Reply:
Suggest this language be:

Cannot have both the docker-py and docker Python packages installed together as they use the same namespace and cause a corrupt installation. Please uninstall both packages, and re-install only the docker-py or docker Python package.

If it's possible to add a single quote without causing a parsing problem, it would be even better as:

Cannot have both the 'docker-py' and 'docker' Python packages installed together as they use the same namespace and cause a corrupt installation. Please uninstall both packages, and re-install only the 'docker-py' or 'docker' Python package.